data/releases/schedule.yaml

```diff
@@ -2,6 +2,14 @@
 # This file helps to populate the /releases page, and is also parsed to find out the
 # latest patch version for a minor release.
 schedules:
+- release: 1.30
+  releaseDate: 2024-04-17
+  next:
+    release: 1.30.1
+    cherryPickDeadline: 2024-05-03
+    targetDate: 2024-05-08
```

Comment:
Are these the updated dates for upcoming May release(s)? Currently on the website [(here)](https://kubernetes.io/releases/patch-releases/#upcoming-monthly-releases), we're suggesting the proposed dates for May 2024 as `2024-05-10` and `2024-05-15`, which differ from the dates in this PR.

Reply:
Hmmm, let me update these then